osu.Game.Rulesets.Tau/Mods/TauModInverse.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using osu.Framework.Graphics;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Tau.Objects;
+using osu.Game.Rulesets.Tau.Objects.Drawables;
+using osu.Game.Rulesets.Tau.UI;
+using osu.Game.Rulesets.UI;
+using osuTK;
+
+namespace osu.Game.Rulesets.Tau.Mods
+{
+    public class TauModInverse : Mod, IApplicableToDrawableHitObject, IApplicableToDrawableRuleset<TauHitObject>
+    {
+        public override string Name => "Inverse";
+        public override string Acronym => "IN";
+        public override ModType Type => ModType.DifficultyIncrease;
+        public override string Description => @"Hit objects will come outside of the playfield.";
+        public override double ScoreMultiplier => 1.2;
+        public override Type[] IncompatibleMods => new[] { typeof(TauModHidden), typeof(TauModFadeIn) };
+
+        public void ApplyToDrawableHitObject(DrawableHitObject drawable)
+        {
+            drawable.ApplyCustomUpdateState += (drawableObject, state) =>
+            {
+                switch (drawableObject)
+                {
+                    case DrawableSliderHead head:
+                        applyInverseToBeat(head, state, -0.484f, -0.984f);
```

Comment:
Why is this treated differently than the regular beats?

Reply:
Since sliders start way earlier than normal beats, we'd have to move the head beat forward a bit

Reply:
Didn't we already fix that problem in the slider head PR?

Reply:
We did not.

Reply:
Then we need to get to the bottom of that, and fix it properly.